### ISSUE TITLE
Server Crash Upon Loading

### ISSUE THREAD

**Issue by TH3GOD88** [Chicken-Bones/EnderStorage#89]
Every time I try to load my server it will not load and I am not sure if it is conflicting with your mod or if something else is going on.
EnderStorage-1.10.2-2.2.1.106-universal.jar
I have also tried an earlier version and it still happens.
Crash Log: https://pastebin.com/j8X2TAE9
Mod List: https://pastebin.com/LSZVDHFi
Log Files:  
[fml-server-2.txt](https://github.com/Chicken-Bones/EnderStorage/files/1221087/fml-server-2.txt)

**Comment by DreadBoy** [comment]
Did you resolve this? It started to happen to me too.

**Comment by DreadBoy** [comment]
I think I figured out the problem so I'm posting this for anybody looking for solution. I checked the method crashlog was talking about and it seems code tries to read certain file "EnderStorage/lock.dat" that should be in your world's directory:

https://github.com/Chicken-Bones/EnderStorage/blob/1.7.2/src/codechicken/enderstorage/api/EnderStorageManager.java#L125

I seems like this file can get corrupted which will cause inbuild gzip reader to trip up, throw unhandled exception and there we are. I restored my lock.dat file from backup and everything's fine.
